src/main/java/backend/controller/UserController.kt

```diff
@@ -145,6 +146,26 @@ class UserController(private val userService: UserService,
         return UserView(user)
     }
 
+    /**
+     * DELETE /user/{id}/
+     * Edits user with given id
+     */
+    @Caching(evict = [(CacheEvict(POSTINGS, allEntries = true)), (CacheEvict(TEAMS, allEntries = true))])
+    @PreAuthorize("isAuthenticated()")

```

Comment:
do we want to allow admins as well to delete users?

Reply:
Right now I'm allowing Admins and the users themselves. We can discuss it, but for now I think we need to have admins on it, just to make sure we shake off all the bugs I wasn't able to see until now